.github/workflows/npm-publish.yml

```
@@ -132,6 +132,7 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.LANCEDB_NPM_REGISTRY_TOKEN }}
         run: |
-          for filename in */*.tgz; do
+          mv */*.tgz .
```

`*/*tgz` does seem a little sketchy. Shouldn't just copying `dist` be sufficient? or does the folder change for some reason?

```suggestion
          mv dist/*.tgz .
```

---

Oh nevermind. Thanks for the explanation!